crates/sui-core/src/db_checkpoint_handler.rs

```diff
@@ -260,6 +260,7 @@ impl DBCheckpointHandler {
         AuthorityStorePruner::prune_objects_for_eligible_epochs(
             &perpetual_db,
             &checkpoint_store,
+            None, //TODO Do we want to open RestIndex
```

Comment:
I ask because this is for object pruning and we don't presently do any object based pruning for this set of indexes.

But lets open it and pass it in either way just to be safe.